tests/_support/Testcases/REST/V1/BaseTicketEditorCest.php

```diff
@@ -175,7 +175,7 @@ protected function create_rsvp_using_rest( Restv1Tester $I, array $variation, ar
 				'_tribe_ticket_capacity'         => '',
 				'_ticket_start_date'             => '2020-01-02 08:00:00',
 				'_ticket_end_date'               => '2050-03-02 20:00:00',
-				'_tribe_ticket_show_not_going'   => 'false',
+				'_tribe_ticket_show_not_going'   => false,
```

Comment:
Should this also be zero, like you did below?
```suggestion
				'_tribe_ticket_show_not_going'   => 0,
```

Reply:
The other ones are actually the odd ones, there is some issue with that particular test where sending `false` is going as ` `. That is why I used 0 to get rid of that issue. But `false` is the correct one to use.

Reply:
Ideally, it should have been fixed within this same scope but as many other PRs for ETP are blocked by this and it's a minor issue impacting this test only, therefore, we have made the decision to go with this workaround for now. We will definitely revisit this later to see why this particular test method is having issues.

I have checked with POSTman directly and the original API works as expected.